lxd/devices.go

```diff
@@ -314,6 +335,26 @@ func deviceNetlinkListener() (chan []string, chan []string, chan device.USBEvent
 	return chCPU, chNetwork, chUSB, chUnix, nil
 }
 
+/*
+ * deviceTaskBalance is used to balance the CPU load across containers running on a host.
```

Comment:
I like this comment! :)

Reply:
yeah, this function was a bit obscure so I thought this was appropriate ;)